### ISSUE TITLE
New what's new card: multiplex

### ISSUE THREAD

**Issue by Rebecca-LM** [CDCgov/prime-simplereport#3557]
Tracking design efforts (content + visuals) for a new card on the What's new page. Once this effort is complete we'll need to add this card to the static site

**Comment by Rebecca-LM** [comment]
@jayna-SkylightDigital for feedback on the Google doc with direction on what we want to communicate about multiplex on the what's new card.

**Comment by Rebecca-LM** [comment]
@kenieh do we have the images we need for this, so I can get it ready this sprint? Context [here](https://docs.google.com/document/d/166844-tbpF-poINLjQG3S0WIgiXKSaSUdhUP2fSPRcA/edit).

**Comment by kenieh** [comment]
@Rebecca-LM haven't designed the images yet since I was waiting for the copy. I wanted the image to compliment the copy. Based on the context you provided, I imagine showing this portion of the test card for the image. Lmk what y'all tink!

<img width="474" alt="Screen Shot 2022-04-13 at 1 32 35 PM" src="https://user-images.githubusercontent.com/72819285/163265599-78dd4f32-b013-4a12-ad44-2e7b4821d276.png">

**Comment by Rebecca-LM** [comment]
@jayna-SkylightDigital [took at stab at the language](https://docs.google.com/document/d/166844-tbpF-poINLjQG3S0WIgiXKSaSUdhUP2fSPRcA/edit) that might work for this card. Let me know what you think.

**Comment by Rebecca-LM** [comment]
@kenieh I think we're ready for design for this one

**Comment by Rebecca-LM** [comment]
@lauraykerr for visibility

**Comment by emmastephenson** [comment]
Vote is to get this in earlier rather than later (so before the multiplex rollout, if need be)
